src/sickle.ts

```diff
@@ -33,6 +33,8 @@ export let closureExternsBlacklist: string[] = [
   'exports',
   'global',
   'module',
+  'WorkerGlobalScope',
```

Comment:
This list needs to be more structured (we can keep hacking on it for now). Here is what I did in clutz:

- fix which files are considered "platform apis" on both sides:
  - lib.d.ts for TS (TSsymbols)
  - closure/externs:COMMON (ClosureSymbols)

- modify lib.d.ts to be a strict subset of closure externs
- enumerate symbols in missingPlatformSymbols = ClosureSymbols \ TSSymbols ( \ is set difference)

clutz now emits everything not in closure externs + missingPlatformSymbols.

Beyond "platform externs", I don't think the tool should be doing any white/black listing, as we can't have any visibility into what those symbols are. It won't scale to keep adding them here.